go/libraries/doltcore/env/environment.go

```diff
@@ -579,3 +581,31 @@ func (dEnv *DoltEnv) GetDefaultRemote() (Remote, errhand.VerboseError) {
 func (dEnv *DoltEnv) GetUserHomeDir() (string, error) {
 	return getHomeDir(dEnv.hdp)
 }
+
+// SetStandardEventAttributes sets local remote urls, and current branch as attributes
+// on an Event
+func (dEnv *DoltEnv) SetStandardEventAttributes(evt *events.Event) {
+	if evt != nil {
+		if len(evt.GetAttribute(eventsapi.AttributeID_LOCAL_REMOTE_URLS)) < 1 {
+			remotes, _ := dEnv.GetRemotes()
+
+			if len(remotes) > 0 {
+				urls := make([]string, len(remotes))
+
+				for _, remote := range remotes {
+					urls = append(urls, remote.Url)
+				}
+
+				joined := strings.Join(urls[:], " ")
+
+				evt.SetAttribute(eventsapi.AttributeID_LOCAL_REMOTE_URLS, strings.Trim(joined, " "))
```

Comment:
I'm not sure you want to capture all the remote urls for every call.  If a push is called, I want to be able to figure out which repo was pushed to.  if multiple remotes are here then I won't be able to distinguish between them.

Reply:
So I considered the AttributeID `LOCAL_REMOTE_URLS` for these "inactive" remotes, and then making an `ACTIVE_REMOTE_URL` AttributeID that is specifically used on CLONE, PUSH, PULL, FETCH, (and maybe REMOTE_ADD ?)... what do you think of something like that?

Also do we care very much about these "inactive" `LOCAL_REMOTE_URLS` that aren't actually being used outside of the commands listed above?